crates/indexer/src/entry.rs

```diff
@@ -144,6 +146,9 @@ fn create_pool(
                 Job::GetStorefronts => get_storefronts::run(&client, handle),
                 Job::GetBidderMetadata => bidder_metadata::get(&client, &bid_map, handle),
                 Job::GetBidderMetadataSolo => bidder_metadata::get_solo(&client, handle),
+                Job::MetadataByCreator(pubkey) => {
+                    metadata::handle_metadata_by_creator(&client, pubkey, handle)
+                },
```

Comment:
This should go below the match arm for `Job::SoloBidsForAuction` for consistency with the enum definition order